spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/plugin/GravitinoDriverPlugin.java

```diff
@@ -51,8 +58,13 @@ public Map<String, String> init(SparkContext sc, PluginContext pluginContext) {
         StringUtils.isNotBlank(metalake),
         String.format(
             "%s:%s, should not be empty", GravitinoSparkConfig.GRAVITINO_METALAKE, metalake));
+    if (conf.contains(GravitinoSparkConfig.GRAVITINO_ENABLE_ICEBERG_SUPPORT)
+        && "true".equals(conf.get(GravitinoSparkConfig.GRAVITINO_ENABLE_ICEBERG_SUPPORT))) {
+      this.supportsIceberg = true;
+      gravitinoDriverExtensions.add(ICEBERG_SPARK_EXTENSIONS);
```

Comment:
If this is false, can we still use Iceberg, or we can use Iceberg with limited functionalities?

Reply:
if this is false, we couldn't use Iceberg at all